tests/test_lanes.py

```diff
@@ -936,6 +937,54 @@ def test_overview_facets(self):
         eq_(known_facets, wl.overview_facets(self._db, some_other_facets))
 
 
+class TestJackpotFacets(DatabaseTest):
+
+    def test_default_facet(self):
+        # A JackpotFacets object defaults to showing only books that
+        # are currently available. Normal facet configuration is
+        # ignored.
+        m = JackpotFacets.default_facet
+
+        default = m(None, JackpotFacets.AVAILABILITY_FACET_GROUP_NAME)
+        eq_(Facets.AVAILABLE_NOW, default)
+
+        # For other facet groups, the class defers to the Facets
+        # superclass. (But this doesn't matter because it's not relevant
+        # to the creation of jackpot feeds.)
+        for group in (Facets.COLLECTION_FACET_GROUP_NAME,
+                      Facets.ORDER_FACET_GROUP_NAME):
+            eq_(m(self._default_library, group),
+                Facets.default_facet(self._default_library, group))
+
+        eq_(Facets.ORDER_SERIES_POSITION, SeriesFacets.DEFAULT_SORT_ORDER)
```

Comment:
I don't get this test. These two are constants but how are they related to the JackpotFacets?

Reply:
They're not; this is a copy-and-paste error. I copied the code from `TestSeriesFacets.test_default_sort_order` and I'm not sure why. Maybe I intended to modify it but then I wrote new code instead.